index.js

```diff
@@ -1,7 +1,7 @@
 const Clownface = require('./lib/Clownface')
 
-function factory ({ dataset, graph, term, value, _context }) {
-  return new Clownface({ dataset, graph, term, value, _context })
+function factory ({ dataset, graph, term, value, factory, _context }) {
+  return new Clownface({ dataset, graph, term, value, factory, _context })
```

Comment:
Doesn't the `factory` variable conflict with the function name?

Reply:
I don't think that's an issue